theme/static/css/hydroshare_core.css

```diff
@@ -983,33 +990,38 @@ li.account p{
 	table-layout: fixed;
 }
 
-.authors-wrapper > a:not(:first-child)::before, .authors-wrapper span:not(:first-child):not(.ui-sortable-helper) a::before {
+.authors-wrapper > a:not(:first-child)::before,
+.authors-wrapper span:not(:first-child):not(.ui-sortable-helper) a::before,
+.authors-wrapper > .ui-state-highlight:not(:first-child)::before,
+.owners-wrapper > a:not(:first-child)::before,
+.separate :not(:first-child)::before {
     content: "";
     width: 1px;
     display: inline-block;
     background-color: #999;
-	height: 22px;
+    height: 1em;
     margin: 0 4px;
     position: relative;
     top: 2px;
 }
 
-.authors-wrapper > a:not(:first-child)::before, .authors-wrapper span:not(:first-child):not(.ui-sortable-helper) a::before,
-.owners-wrapper > a:not(:first-child)::before, .separate :not(:first-child)::before {
-    content: "";
-    width: 1px;
-    display: inline-block;
-    background-color: #999;
-    height: 1em;
-    margin: 0 4px;
-    position: relative;
-    top: 2px;
+.authors-wrapper > .ui-state-highlight:not(:first-child)::before {
+	margin: 0 -6px;
 }
 
 .authors-wrapper.sortable span:last-child a {
     margin-right: 10px;
 }
 
+.authors-wrapper .ui-state-highlight {
+	width: 100px;
+	height: 1em;
+	display: inline-block;
+	margin-left: 8px;
+	margin-right: 4px;
+	padding: 0;
+}
```

Comment:
It's been many years, since I coded CSS in depth, so I was looking around and came across this guide. If you agree with its advice, maybe check out the margin section https://gist.github.com/basham/2175a16ab7c60ce8e001

Reply:
In this case this component needs the units in 'em' because it is specifically designed to accommodate this element's font-size as the height. It can be reused like this anywhere on the site and it would keep using the correct value since 'em' refers to the font-size of the element.